migrations/Version202108190925463772_taoLti.php

```diff
@@ -0,0 +1,57 @@
+<?php
+
+declare(strict_types=1);
+
+namespace oat\taoLti\migrations;
+
+use Doctrine\DBAL\Schema\Schema;
+use oat\tao\scripts\tools\migrations\AbstractMigration;
+use oat\tao\model\search\SearchProxy;
+
+final class Version202108190925463772_taoLti extends AbstractMigration
```

Comment:
Will same changes applied on fresh installation?
TO not forget 
- [ ] update dependency from tao-core with newer version

Reply:
Not applied when running install command. But executed after running taoUpdate.php

Reply:
Done